lib/root_loader.rb

```diff
@@ -33,6 +35,22 @@ def load account, filename
 
         loader = Loader::Orchestrate.new policy_version
         loader.load
+
+        accepted_roles = loader.new_roles.select do |role|
+          %w(user host).member?(role.kind)
+        end
+
+        created_roles = acceppted_roles.each_with_object({}) do |role, memo|
```

Comment:
Typo here: `acceppted`. I would have thought this would break when testing, have you tried this out?

Reply:
Good catch! It broke while testing.